src/Elastic.Apm/Api/IError.cs

```diff
@@ -41,4 +50,43 @@ public interface IError
 		/// </summary>
 		string TransactionId { get; }
 	}
+
+	/// <summary>
+	/// Represents a log line which is captured as part of an APM error.
+	/// </summary>
+	public class ErrorLog
+	{
+		public ErrorLog(string message)
+			=> Message = message;
+
+		/// <summary>
+		/// The severity of the record.
+		/// </summary>
+		[MaxLength]
+		public string Level { get; set; }
+
+		/// <summary>
+		/// The name of the logger instance used.
+		/// </summary>
+		[JsonProperty("logger_name")]
+		[MaxLength]
+		public string LoggerName { get; set; }
+
+		/// <summary>
+		/// The additionally logged error message.
+		/// </summary>
+		public string Message { get; set; }
+
+		/// <summary>
+		/// A parametrized message. E.g. 'Could not connect to %s'. The property message is still required, and should be equal
+		/// to the param_message, but with placeholders replaced. In some situations the param_message is used to group errors
+		/// together.
+		/// The string is not interpreted, so feel free to use whichever placeholders makes sense in the client languange."
+		/// </summary>
+		[JsonProperty("param_message")]
+		[MaxLength]
+		public string ParamMessage { get; set; }
+
+		public List<CapturedStackFrame> StackTrace { get; set; }
```

Comment:
Does this need `[JsonProperty("stacktrace")]` ?

Reply:
Yes - added.